RabbitMQ, GoogleTest, SimpleAmpqClient/GoogleTest.md

```diff
@@ -0,0 +1,30 @@
+#GoogleTest.md
+
+##GoogleTest 
+
+Testing the telemetry test tool and Hermes is done with googletest and google mock 
+
+the gmock and gtest header files need to be in your include path `/usr/local/include` or `/usr/include`.
+
+You must compile the library that has the definitions yourself.
+Meaning you need a `libgmock.a` archive file in the test directory in order to compile the tests.
+
+To create a `libgmock.a` file in a Linux system follow these instructions in an arbitrary directory.
+
+1. Get the googletest and googlemock source code:
+    `git clone https://github.com/google/googletest.git`
+
+2. Go into the cloned directory:
+    `cd googletest/`
+
+3. Compile googletest:
+    `g++ -isystem googletest/include/ -Igoogletest -isystem googlemock/include/ -Igooglemock -pthread -c googletest/src/gtest-all.cc`
+
+4. Compile googlemock:
+    `g++ -isystem googletest/include/ -Igoogletest -isystem googlemock/include/ -Igooglemock -pthread -c googlemock/src/gmock-all.cc`
+
+5. Collect both the resulting .o files into an archive file:
+    `ar -rv libgmock.a gtest-all.o gmock-all.o`
+
+Congratulations! You now have a `libgmock.a` archive file.
+
```

Comment:
Let's add a sixth step: 
`6. Both gmock and the gtest folders located in googletest/googlemock/include/ and googletest/googletest/include/ need to be placed in the /usr/local/include system directory.`